src/wmnds/components/file-download/_file-download.scss

```
@@ -8,7 +8,9 @@
 
   &__icon {
     width: 2.5rem;
+    min-width: 2.5rem;
     height: 2.5rem;
+    min-height: 2.5rem;
```

Would it not be better to use flex to enforce size here?
```suggestion
  &__icon {
    width: 2.5rem;
    height: 2.5rem;
    flex-grow: 0;
    flex-shrink: 0;
```

---

or
```suggestion
  &__icon {
    width: 2.5rem;
    height: 2.5rem;
    flex: 0 0 2.5rem;
```

---

There is another way of doing this which would probably have better browser support.....
Make the **.wmnds-file-download** `display:block;` instead of `display:flex`.

Then on both of the child items, make them `display: inline-block; vertical-align:middle;`.

And finally on the **.wmnds-file-download__desc** make sure it has these CSS attributes:
```css
padding: .75rem;
display: inline-block;
vertical-align: middle;
width: calc(100% - 2.5rem - .75rem);
```

The calc on the width, takes off the padding(.75rem) and takes off the icon width (2.5rem) so it will fit in correctly.

Also, is there a way we can replicate the issue/code in the issue screenshot so we can test that it's fixed!

---

https://beta.wmnetwork.co.uk/resources-and-support/resources-and-support-for-businesses/